tests/integration/synapseclient/core/test_external_storage.py

```diff
@@ -304,7 +304,7 @@ async def test_sts_external_storage_location(self) -> None:
                     Bucket=bucket_name,
                     Key=remote_key,
                 )
-            assert "Access Denied" in str(ex_cm.value)
+            assert "AccessDenied" in str(ex_cm.value)
```

Comment:
Interesting, what changed about this?

Reply:
It just looks like AWS changed the error returned 🤷🏻